### ISSUE TITLE
Thread Panic on Execution

### ISSUE THREAD

**Issue by jp-u** [lukehsiao/poetry-udeps#2]
I've encountered an error while trying to run this on a repo:

```sh
> poetry-udeps

thread 'main' panicked at <USER_HOME>/.cargo/registry/src/index.crates.io-6f17d22bba15001f/poetry-udeps-0.2.2/src/lib.rs:98:22:
called `Option::unwrap()` on a `None` value
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

With the backtrace:
```
stack backtrace:
   0: rust_begin_unwind
             at /rustc/cc66ad468955717ab92600c770da8c1601a4ff33/library/std/src/panicking.rs:595:5
   1: core::panicking::panic_fmt
             at /rustc/cc66ad468955717ab92600c770da8c1601a4ff33/library/core/src/panicking.rs:67:14
   2: core::panicking::panic
             at /rustc/cc66ad468955717ab92600c770da8c1601a4ff33/library/core/src/panicking.rs:117:5
   3: poetry_udeps::get_dependencies
   4: poetry_udeps::run
   5: poetry_udeps::main
note: Some details are omitted, run with `RUST_BACKTRACE=full` for a verbose backtrace.
```

**Comment by lukehsiao** [comment]
Is the repo public so I can reproduce?

**Comment by lukehsiao** [comment]
Looking at the code, it seems this is a bug in parsing your pyproject.toml. In particular, the dev dependencies. If it's possible to share that, I can probably fix this pretty quick.

**Comment by lukehsiao** [comment]
Nevermind, I see the issue: we assume that a project has dev dependencies, when it might not.

For example

```
[tool.poetry]
name = "tmp"
version = "0.1.0"
description = ""
readme = "README.md"

[tool.poetry.dependencies]
python = "^3.12"


[build-system]
requires = ["poetry-core"]
build-backend = "poetry.core.masonry.api"
```

I'll release a fix momentarily.

**Comment by lukehsiao** [comment]
This should be fixed in `v0.2.3`. Please give it a try.
